frontend/src/scenes/groups/cards/GroupDashboardCard.tsx

```diff
@@ -19,9 +19,18 @@ function GroupDetailDashboard({
     groupData: Group
 }): JSX.Element {
     const { groupTypeName } = useValues(groupLogic)
-    const { setProperties, setLoadLayoutFromServerOnPreview } = useActions(
-        dashboardLogic({ id: groupTypeDetailDashboard })
-    )
+    const dashboardLogicProps = {
+        id: groupTypeDetailDashboard,
+        variables: {
+            group_key: {
+                variableId: 'group_key',
```

Comment:
What trickery is this :D

Backend side variables are referenced by id and then looked up from the saved ones in the [map_stale_to_latest](https://github.com/PostHog/posthog/blob/9235f4abce9adcbd10bb5df43f5e0cfc0d0f0a43/posthog/api/insight_variable.py#L43) method. If you follow how the method is called, you'll see that `InsightVariable.objects.filter(team=self.team).all()` is passed in as `latest_variables` param. Unless there's more to it that I'm unaware of, I think the code paths using this would break.